projects/fal/src/fal/app.py

```diff
@@ -64,15 +65,33 @@ def initialize_and_serve():
     return fn
 
 
+PART_FINDER_RE = re.compile(r"[A-Z][a-z]*")
+
+
+def _to_fal_app_name(name: str) -> str:
+    # Convert MyGoodApp into my-good-app
+    return "-".join(part.lower() for part in PART_FINDER_RE.findall(name))
+
+
 @mainify
 class App(fal.api.BaseServable):
     requirements: ClassVar[list[str]] = []
     machine_type: ClassVar[str] = "S"
-    host_kwargs: ClassVar[dict[str, Any]] = {}
+    host_kwargs: ClassVar[dict[str, Any]] = {
+        "_scheduler": "nomad",
+        "_scheduler_options": {
+            "storage_region": "us-east",
```

Comment:
does this mean fast lambda cluster always?

Reply:
no. it means anything in east which has other stuff too (like a6000 machines)